src/redux/watchList/reducer.js

```diff
@@ -0,0 +1,61 @@
+import { FETCH_WATCHLIST_SUCCESS, POST_WATCHLIST_SUCCESS } from './actionCreator';
+
+const watchlist1 = {
+  id: 1,
+  name: 'A share',
+  stocks: [],
+};
+
+const watchlist2 = {
+  id: 2,
+  name: 'NSDQ',
+  stocks: [],
+};
+
+const initalState = {
+  watchlists: [watchlist1, watchlist2],
+  stocks: [
+    {
+      id: 1,
+      stockName: 'MSFT',
+      curPrice: 141.21,
+      fluctuation: 0.23,
+      dealAmount: '1.2M',
+    },
+    {
+      id: 2,
+      stockName: 'AMZN',
+      curPrice: 1208.2,
+      fluctuation: -0.06,
+      dealAmount: '609k',
+    },
+    {
+      id: 3,
+      stockName: 'AAPL',
+      curPrice: 250.7,
```

Comment:
do not use custom abbreviations, it could be currPrice, currentPrice, actualPrice whatever, but `cur` is strange